console.js

```diff
@@ -0,0 +1,37 @@
+const repl = require('repl'),
+  fs = require('fs'),
+  pjson = require('./package.json'),
+  orm = require('./app/orm'),
+  testFolder = './tests/';
+
+const convertFunctionToAsync = f => {
+  return async (...args) => {
+    const result = await f(...args);
+    console.log(result);
+  };
+};
+
+const convertObjectFunctionsToAsync = serviceMethods => {
+  const asyncServiceMethods = {};
+  Object.keys(serviceMethods).forEach(key => {
+    asyncServiceMethods[key] = convertFunctionToAsync(serviceMethods[key]);
+  });
+  return asyncServiceMethods;
+};
+
+orm.init().then(() => {
+  const replServer = repl.start({
+    prompt: `${pjson.name}> `
+  });
+  const servicesPath = './app/services/';
+  fs.readdir(servicesPath, (err, files) => {
+    files.forEach(file => {
+      const serviceMethods = require(`${servicesPath}${file}`);
+      const asyncServiceMethods = convertObjectFunctionsToAsync(serviceMethods);
+      replServer.context[`${file.split('.')[0]}Service`] = asyncServiceMethods;
+      replServer.context.orm = orm;
```

Comment:
This should be done outside the `forEach` statement. Also can be set outside directory reading.